metadata-ingestion/src/datahub/ingestion/run/pipeline_config.py

```diff
@@ -50,7 +50,7 @@ class FlagsConfig(ConfigModel):
     )
 
     generate_browse_path_v2_dry_run: bool = Field(
-        default=True,
+        default=False,
```

Comment:
why doesn't this change impact any golden files?

Reply:
It most certainly does lol